### DISCUSSION TITLE
Welcome to go-rest-api Discussions!

### DISCUSSION THREAD

**Discussion by qiangxue** [qiangxue/go-rest-api#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kolkov** [comment]
Thank a lot for this starter. I use this template for multiple private projects and one open source project. But I need contributers. :)
https://github.com/goprerender/prerender

**Comment by frkntplglu** [comment]
Hi, first of all, I appreciate your effort and good job. I saw that you created Repository interface in your repository file however as far as I know interfaces should be created on the consumer side, not on the producer. Why did you choose to create in producer?
